kube-controllers/pkg/controllers/node/labels.go

```diff
@@ -197,21 +196,21 @@ func (c *nodeLabelController) syncNodeLabels(node *v1.Node) {
 		// Set the annotation to the correct values.
 		bytes, err := json.Marshal(node.Labels)
 		if err != nil {
-			log.WithError(err).Errorf("Error marshalling node labels")
+			logrus.WithError(err).Errorf("Error marshalling node labels")
 			return
 		}
 		calNode.Annotations[nodeLabelAnnotation] = string(bytes)
 
 		// Update the node in the datastore.
 		if needsUpdate {
 			if _, err := c.client.Nodes().Update(context.Background(), calNode, options.SetOptions{}); err != nil {
-				log.WithError(err).Warnf("Failed to update node, retrying")
+				logrus.WithError(err).Warnf("Failed to update node, retrying")
 				time.Sleep(retrySleepTime)
 				continue
 			}
-			log.WithField("node", node.ObjectMeta.Name).Info("Successfully synced node labels")
+			logrus.WithField("node", node.ObjectMeta.Name).Info("Successfully synced node labels")
 		}
 		return
 	}
-	log.Errorf("Too many retries when updating node")
+	logrus.Errorf("Too many retries when updating node")
```

Comment:
Do we have a preference or reason to use `log` vs `logrus` or vice-versa?

Reply:
For _this_ patch I took the approach of minimising changes.  I'm considering following up with a `gopatch` PR that makes it all uniform.

Reply:
FWIW, yes, I have a preference.  I prefer `logrus` because that's the module name.  (And in the RS repo we've largely standardised on that.)

Reply:
I prefer logrus too but in int_dataplane there was one instance of logrus and hundreds of `log` and I expected merge conflicts galore if I just changed that to logrus